onnxruntime/python/tools/transformers/models/llama/convert_to_onnx.py

```diff
@@ -539,6 +540,23 @@ def remove_existing_files(output_path: str):
             logger.warning(f"Removed {filepath}")
 
 
+def optimize_optimum(config: PretrainedConfig, args):
```

Comment:
```suggestion
def optimize_optimum(config: AutoConfig, args):
```

Is `PretrainedConfig` needed? The `config` is passed to `use_group_query_attention`, which expects `AutoConfig`.

Reply:
Just fixed - I believe AutoConfig derives from PretrainedConfig, but using AutoConfig all the way through is much cleaner.